lib/ripper-tags/parser.rb

```diff
@@ -382,12 +388,15 @@ def on_class_eval(name, body)
       @namespace.pop
     end
 
+    def on_args(*args)
+      process(args)
```

Comment:
I'm worried about the potential negative implications of this. Have you tried running ripper-tags from this branch against a large Ruby codebase such as Rails?

Reply:
Yes, great idea. After my subsequent changes, I've confirmed that the tags files have no changes between master and this branch.